systemtest/elasticsearch.go

```diff
@@ -93,11 +93,14 @@ func CleanupElasticsearch(t testing.TB) {
 }
 
 func cleanupElasticsearch() error {
-	_, err := Elasticsearch.Do(context.Background(), &esapi.IndicesDeleteDataStreamRequest{Name: []string{
-		"traces-apm*",
-		"metrics-apm*",
-		"logs-apm*",
-	}}, nil)
+	_, err := Elasticsearch.Do(context.Background(), &esapi.IndicesDeleteDataStreamRequest{
+		Name: []string{
+			"traces-apm*",
+			"metrics-apm*",
+			"logs-apm*",
+		},
+		ExpandWildcards: "all,hidden",
```

Comment:
Is `hidden` necessary when setting `all`? See https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-update-settings.html

Reply:
For delete data stream, the docs https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-delete-data-stream.html indicate that the valid values are `all,hidden`, `open,closed` and `none`, while the default is `open`, which isn't even a valid value. Not sure if my brain parses the docs right, because it confuses me a little bit. I haven't tried `open,hidden` for this yet. Will try and see if it works the same.

Reply:
When I look at the docs again, I think it meant `all` == `hidden`. Updated PR to use `all` only.